src/view/kc-webcam.js

```diff
@@ -5,11 +5,20 @@ export class KCWebCam extends LitElement {
     super()
 
     this.webcamIndex = 0;
+    this.statefulThing = 0;
     this.getCameras().then((videoDevices) => {
       this.setUpCamera(videoDevices[this.webcamIndex])
     })
   }
 
+  firstUpdated(changedProps) {
+    console.log('i should display once')
+  }
+
+  updated(changedProps) {
+    console.log('i !!!should!!! display lots')
+  }
+
   static get properties () {
```

Comment:
I thought the stateful stuff was to be declared here. You don't have to call update, if you declare statefulThing in the props. The render method will be fire at the end of a cycle if one of the properties has been changed.